components/motor/gpio/motor_encoder.go

```diff
@@ -256,13 +257,41 @@ func (m *EncodedMotor) goForMath(ctx context.Context, rpm, revolutions float64)
 // If revolutions is 0, this will run the motor at rpm indefinitely
 // If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
 func (m *EncodedMotor) GoFor(ctx context.Context, rpm, revolutions float64, extra map[string]interface{}) error {
-	m.opMgr.CancelRunning(ctx)
```

Comment:
...wasn't this line important? If we don't stop the old operation before starting the new one, could anything bad happen?

Reply:
No, your explanation that `New` cancels anything else is perfect! Let's keep the line removed, so I don't ask this same question next time someone wants to remove it.